omod/src/main/java/org/openmrs/module/webservices/rest/web/resource/FlagsResource.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+package org.openmrs.module.webservices.rest.web.resource;

```

Comment:
Fix package (see [this](https://github.com/openmrs/openmrs-module-patientflags/pull/3/files#r66771305) comment).